examples/nodejs-simple/src/index.js

```diff
@@ -13,9 +13,7 @@
 // limitations under the License.
 
 const AgonesSDK = require('@google-cloud/agones-sdk');
-const util = require('util');
-
-const sleep = util.promisify(setTimeout);
+const {setTimeout} = require('timers/promises');
```

Comment:
This will require using Node.js 16, we could revert if we want the examples to remain backwards compatible

Reply:
I'd be more concerned if the SDK code required the new version but I think it's ok for the example code to use the new version.